src/app/pages/HomePage/index.test.tsx

```diff
@@ -117,6 +121,44 @@ describe('Home Page', () => {
     });
   });
 
+  describe('Lazy Loading', () => {
+    // const { BANNER } = VISUAL_STYLE;
+    // const { NORMAL } = VISUAL_PROMINENCE;
+    // const messageBannerImages = kyrgyzHomePageData.curations
+    //   .filter(
+    //     ({ visualStyle, visualProminence }) =>
+    //       visualStyle === BANNER && visualProminence === NORMAL,
+    //   )
+    //   .flatMap(curation => {
+    //     return curation.summaries?.map(summary => summary.imageUrl);
+    //   });
+
+    it('Only the first image and message banner on the homepage are not lazy loaded, but all others are', () => {
+      render(<HomePage pageData={homePageData} />, {
+        service: 'kyrgyz',
+      });
+
+      const messageBannerImages: string[] = [];
+      document
+        .querySelectorAll(`[data-testid^="message-banner"] img`)
+        .forEach(image =>
+          messageBannerImages.push(image.getAttribute(`src`) || ''),
+        );
+
+      const imageList = document.querySelectorAll('img');
+      imageList.forEach((image, index) => {
+        if (index === 0) {
+          expect(image.getAttribute('loading')).toBeNull();
+        } else {
+          const src = image.getAttribute('src') || '';
+          if (!messageBannerImages.includes(src)) {
+            expect(image.getAttribute('loading')).toBe('lazy');
+          }
```

Comment:
Can we tweak this slightly, to ensure that we test that the message banner images are not lazy loaded:

```suggestion
       const src = image.getAttribute('src') || '';

        if (index === 0 || messageBannerImages.includes(src)) {
          expect(image.getAttribute('loading')).toBeNull();
        }
          if (!messageBannerImages.includes(src)) {
            expect(image.getAttribute('loading')).toBe('lazy');
```

Reply:
Maybe don't commit my suggestion, as I think the brackets are all messed up! Try this locally, and run the tests to ensure they are still passing.